### ISSUE TITLE
#error emitted by code generators isn't shown

### ISSUE THREAD

**Issue by Bosch-Eli-Black** [dotnet/roslyn#60447]
**Version Used**: 
VS 17.2.0 Preview 2.0

**Overview**
When a code generator emits code that contains the `#error` preprocessor directive, the error message isn't shown within Visual Studio or while building.

**Steps to Reproduce**:

1. Write an analyzer that emites code like this:
```c#
#error Source generator encountered an issue
``` 

**Expected Behavior**:

The error message shows up while building and inside Visual Studio.

**Actual Behavior**:

The error message is not shown.

**Comment by Bosch-Eli-Black** [comment]
For that matter, errors in the generated C# code aren't shown, either.

For example, if the analyzer emits C# code that's like this, no error is shown in VS (or while compiling, I think):
```c#
class class class {;
```

**Comment by Bosch-Eli-Black** [comment]
Just updated from VS 17.2.0 Preview 2.0 to Preview 2.1 and now this is working, so closing this 🙂
